build/NugetVersionDeprecator/NugetVersionDeprecator.csproj

```diff
@@ -19,11 +19,11 @@
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="NuGet.Protocol" Version="6.6.1" />
-    <PackageReference Include="Octokit" Version="7.1.0" />
-    <PackageReference Include="RestSharp" Version="110.2.0" />
-    <PackageReference Include="RestSharp.Serializers.NewtonsoftJson" Version="110.2.0" />
-    <PackageReference Include="YamlDotNet" Version="13.1.1" />
+    <PackageReference Include="NuGet.Protocol" Version="6.9.1" />
+    <PackageReference Include="Octokit" Version="10.0.0" />
+    <PackageReference Include="RestSharp" Version="110.2.1-alpha.0.16" />
```

Comment:
Do we really want to use pre-release versions here?

Reply:
No, no we don't. 😄